pkg/resource/deploy/step_generator.go

```diff
@@ -883,13 +858,122 @@ func (sg *stepGenerator) continueStepsFromRefresh(event ContinueResourceRefreshE
 			sg.deployment.newPlans.set(urn, newResourcePlan)
```

Comment:
Is the preceding comment and code block (about how imports have no diff) true any more? If not, how does this affect plans (if at all)?